app/models/property.rb

```diff
@@ -0,0 +1,12 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+class Property
+
+include DataMapper::Resource
+
+property :id,       Serial
+property :name,     String
```

Comment:
Let's discuss having all the information from the user story here (location, etc). Otherwise, we will have multiple pull requests per user story.